src/libraries/Microsoft.PowerFx.Core/Lexer/WhitespaceToken.cs

```diff
@@ -21,9 +21,9 @@ public WhitespaceToken(string value, Span span)
             Value = value;
         }
```

Comment:
Not your change, but why isn't this in Lexer/Tokens?

Reply:
Good point. Note that the namespace is also different (matches the location). Should I move it?

Reply:
Yeah, I think so.